src/templates/chapter.js

```diff
@@ -56,13 +56,16 @@ const ChapterTemplate = ({ data: { mdx: chapter } }) => {
     `${chapter.frontmatter.editor.startingCode}`,
   );
   const [showOutput, setShowOutput] = useState(false);
+  const [buttonClicked, setButtonClicked] = useState(false);
+  var [editorHeight, setEditorHeight] = useState(`calc(100vh - (210px + 40px))`);
   useEffect(() => {
     monaco
       .init()
       .then(monacoInstance => {
         monacoInstance.editor.defineTheme('myCustomTheme', {
           base: 'vs-dark',
           inherit: true,
+          automaticLayout: true,
```

Comment:
what does this option do?

Reply:
This option is used so that the editor covers only the remaining screen to prevent overflow while typing code in the editor.